### ISSUE TITLE
Tests fail on go1.19

### ISSUE THREAD

**Issue by fyrchik** [nspcc-dev/neofs-sdk-go#316]
```
goroutine 19 [running]:
testing.tRunner.func1.2({0xac81c0, 0xcf20d0})
        /usr/lib/go/src/testing/testing.go:1396 +0x24e
testing.tRunner.func1()
        /usr/lib/go/src/testing/testing.go:1399 +0x39f
panic({0xac81c0, 0xcf20d0})
        /usr/lib/go/src/runtime/panic.go:884 +0x212
crypto/elliptic.panicIfNotOnCurve({0xd00bc8?, 0x121b590?}, 0x80?, 0x7?)
        /usr/lib/go/src/crypto/elliptic/elliptic.go:182 +0xa5
crypto/elliptic.Marshal({0xd00bc8, 0x121b590}, 0xc000157628?, 0xc0001575c8?)
        /usr/lib/go/src/crypto/elliptic/elliptic.go:75 +0x31
github.com/nspcc-dev/neofs-sdk-go/crypto/ecdsa.Signer.Sign({{{0xd00bc8, 0x121b590}, 0xc000166f20, 0xc000166f40}, 0xc000166f00}, {0xc0001ae160?, 0x0?, 0xc0001ae160?})
        /home/dzeta/repo/neofs-sdk-go/crypto/ecdsa/signer.go:34 +0x116
github.com/nspcc-dev/neofs-sdk-go/crypto.(*Signature).Calculate(0xc000157868, {0xcfe910, 0xc0001e7b60}, {0xc0001ae160?, 0x19?, 0x19?})
        /home/dzeta/repo/neofs-sdk-go/crypto/signature.go:60 +0x58
github.com/nspcc-dev/neofs-sdk-go/session.(*commonData).sign(0xc000157aa8, {{{0xd00bc8, 0x121b590}, 0xc000166f20, 0xc000166f40}, 0xc000166f00}, 0x1?)
        /home/dzeta/repo/neofs-sdk-go/session/common.go:164 +0x1a7
github.com/nspcc-dev/neofs-sdk-go/session.(*Object).Sign(...)
        /home/dzeta/repo/neofs-sdk-go/session/object.go:165
github.com/nspcc-dev/neofs-sdk-go/pool.TestSessionCache_GetUnmodifiedToken(0xc0002861a0?)
        /home/dzeta/repo/neofs-sdk-go/pool/cache_test.go:31 +0x330
testing.tRunner(0xc0002861a0, 0xc30420)
        /usr/lib/go/src/testing/testing.go:1446 +0x10b
created by testing.(*T).Run
        /usr/lib/go/src/testing/testing.go:1493 +0x35f
FAIL    github.com/nspcc-dev/neofs-sdk-go/pool  0.009s
```

**Comment by roman-khimov** [comment]
Same thing as nspcc-dev/neofs-crypto#15.

**Comment by carpawell** [comment]
Solved in #319.
